apps/www/_blog/2023-02-16-case-study-happyteams.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: HappyTeams unlocks better performance and reduces cost with Supabase
+description: How a bootstrapped startup migrated from Heroku to Supabase in 30 minutes and never looked back
+author: rory_wilding
+image: 2023-02-16-case-study-happyteams/case-study-happyteams.png
+thumb: 2023-02-16-case-study-happyteams/case-study-happyteams.png
+tags:
+  - case-study
+date: '2023-02-16'
+toc_depth: 3
+---
+
+## The Challenge 
+
+> Finding a painless and hands-free database experience to support growth
+
+HappyTeams is a growing startup with an ambitious vision. The company is bootstrapped, so technical resources are scarce. Devs need to do more with less. This means staying laser-focused on delivering features, rather than database timeouts or performance issues.
```

Comment:
```suggestion
[HappyTeams](https://happyteams.io/) is a growing startup with an ambitious vision. The company is bootstrapped, so technical resources are scarce. Devs need to do more with less. This means staying laser-focused on delivering features, rather than database timeouts or performance issues.
```